src/main/java/cholog/wiseshop/api/campaign/controller/CampaignController.java

```diff
@@ -45,4 +46,10 @@ public ResponseEntity<List<ReadCampaignResponse>> readAllCampaign() {
         var response = campaignService.readAllCampaign();
         return ResponseEntity.status(HttpStatus.OK).body(response);
     }
+
+    @GetMapping("/member")

```

Comment:
url path는 복수로 쓰는거에 대해 어떻게 생각해?
```suggestion
    @GetMapping("/members")

```